doc/deoplete.txt

```diff
@@ -154,6 +154,93 @@ ignore_case
 ignore_sources
 		It is a dictionary to decide ignore source names.
 		The key is filetype and the value is source names list.
+		Default value: {}
+
+						*deoplete-options-attrs_order*
+attrs_order
```

Comment:
`attrs_order` should be `converter_reorder_attr` specific var like `delimiters` in `converter_auto_delimiter`.